### PR TITLE
Documentation and README changes

### PR DESCRIPTION
**Pull request by nskh**
Modified the homepage of the documentation to point to the tutorials with more emphasis. I also added a comment in `conf.py` that would enable a mock import for traci, which helps but doesn't fully resolve our autodoc issues. (I'll add a full PR for autodocs with I figure them out).  Resolves https://github.com/berkeleyflow/flow-devel/issues/76.

### CODE REVIEW

**Comment by eugenevinitsky** [comment]
I think it might also be helpful to add a sidebar link to the tutorials; i.e. just like it says 
image
it might be worthwhile to have one of those segments say tutorials. Increases the likelihood of them finding it.

**Comment by nskh** [comment]
Sounds good - done. `make html` on this branch to see the result.

**Comment by nskh** [comment]
Just pulled in Aboudy's setup instructions updated in https://github.com/berkeleyflow/flow-devel/pull/78

File: docs/source/conf.py [hunk 1]
```diff
@@ -37,6 +37,14 @@
 napoleon_use_param = False
 napoleon_use_ivar = True
 
+# This prevents some errors during the autodoc building process, since
+# errors occur when importing traci and sumolib. A mock import for traci
+# enables some more autodocs to be built, as flow.core.vehicles, but
+# mock imports for sumolib (another module that errors) leads to build failures
+# entirely.
+
+# autodoc_mock_imports = ["traci"]
```
Comment (cathywu) [inline docs/source/conf.py]: why is this commented out?

File: docs/source/index.rst [hunk 2]
```diff
@@ -6,10 +6,10 @@
 Welcome to Flow
 ================
 
-Flow is a computational framework for deep RL and control experiments for traffic microsimulation.
+`Flow <https://berkeleyflow.github.io>`_ is a computational framework for deep RL and control experiments for traffic microsimulation. Visit `our website <https://berkeleyflow.github.io>`_ for more information. 
 
 Flow is a work in progress - input is welcome. Available documentation is limited for now. 
-Tutorials in iPython notebook format are available through `our GitHub <https://github.com/berkeleyflow/flow/tree/master/tutorials/exercises>`_.
+Tutorials `are available <https://github.com/berkeleyflow/flow/tree/master/tutorials/exercises>`_ in iPython notebook format.
```
Comment (cathywu) [inline docs/source/index.rst]: I like the new text! Can we hyperlink "Tutorials" instead?

File: docs/source/index.rst [hunk 3]
```diff
@@ -6,10 +6,10 @@
 Welcome to Flow
 ================
 
-Flow is a computational framework for deep RL and control experiments for traffic microsimulation.
+`Flow <https://berkeleyflow.github.io>`_ is a computational framework for deep RL and control experiments for traffic microsimulation. Visit `our website <https://berkeleyflow.github.io>`_ for more information.
```
Comment (cathywu) [inline docs/source/index.rst]: 👍

File: docs/source/index.rst [hunk 4]
```diff
@@ -18,11 +18,11 @@ Tutorials in iPython notebook format are available through `our GitHub <https://
 
    intro
    flow_setup
+   Tutorials <https://github.com/berkeleyflow/flow/tree/master/tutorials/>
```
Comment (cathywu) [inline docs/source/index.rst]: 💯

File: docs/source/conf.py (hunk 1 above)
Comment (nskh) [reply docs/source/conf.py]: I think that was leftover from a previous build of the docs that was failing for some reason - I'll uncomment it.

File: docs/source/index.rst (hunk 2 above)
Comment (nskh) [reply docs/source/index.rst]: Addressed.

**Comment by cathywu** [comment]
Thanks a lot for addressing the comments! I already approved in the previous review, so feel free to merge when you are comfortable!

**Pull request merged**